automl/cloud-client/vision_classification_create_model.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def create_model(project_id, dataset_id, display_name):
+    """Create a model."""
+    # [START automl_vision_classification_create_model]
+    from google.cloud import automl
+
+    # TODO(developer): Uncomment and set the following variables
+    # project_id = 'YOUR_PROJECT_ID'
+    # dataset_id = 'YOUR_DATASET_ID'
+    # display_name = 'YOUR_MODEL_NAME'
+
+    client = automl.AutoMlClient()
+
+    # A resource that represents Google Cloud Platform location.
+    project_location = client.location_path(project_id, 'us-central1')
+    # Leave model unset to use the default base model provided by Google
+    metadata = automl.types.ImageClassificationModelMetadata()
```

Comment:
let's add train budget for all (possible) model creation samples

Reply:
it is an optional field, I'm not sure it is worth adding.